api/src/identity-access-management/infrastructure/repositories/anonymous-user-token.repository.js

```diff
@@ -0,0 +1,29 @@
+import crypto from 'node:crypto';
+
+import { config } from '../../../shared/config.js';
+import { temporaryStorage } from '../../../shared/infrastructure/key-value-storages/index.js';
+
+const anonymousUserTokenTemporaryStorage = temporaryStorage.withPrefix('anonymous-user-token:');
```

Comment:
Suggestion pour que les valeurs soient plus faciles à trouver dans Redis et pour être cohérent avec les autres stockages temporaires dans Redis : 
```suggestion
const anonymousUserTokenTemporaryStorage = temporaryStorage.withPrefix('anonymous-user-tokens:');
```